library/proc_macro/src/lib.rs

```diff
@@ -1510,15 +1510,23 @@ pub mod tracked_env {
 }
 
 /// Tracked access to additional files.
-#[unstable(feature = "track_path", issue = "99515")]
+#[unstable(feature = "proc_macro_tracked_path", issue = "99515")]
 pub mod tracked_path {
+    use std::path::{Path, PathBuf};
 
-    /// Track a file explicitly.
+    /// Track a file as if it was a dependency.
+    ///
+    /// The file is located relative to the current file where the proc-macro
+    /// is used (similarly to how modules are found). The provided path is
+    /// interpreted in a platform-specific way at compile time. So, for
+    /// instance, an invocation with a Windows path
+    /// containing backslashes `\` would not compile correctly on Unix.
```

Comment:
I find the note on windows/unix paths quite confusing. Who is that note for? What user error is it trying to prevent?